backend/vaa-strapi/src/util/acl.type.ts

```diff
@@ -0,0 +1,46 @@
+import type { factories } from '@strapi/strapi';
+import type { Strapi } from '@strapi/types';
+
+// Strapi 5: Hopefully, these types can be removed
```

Comment:
These types add a bit of rigour to the ACL functions but hopefully we can get rid of these when updating to Strapi 5, which should also allow us to use the types for the Content Types.